coops-ui/tests/unit/Directors.spec.ts

```diff
@@ -25,7 +25,7 @@ describe('Directors.vue', () => {
     store.state.corpNum = 'CP0001191'
 
     // GET directors
-    sinon.stub(axios, 'get').withArgs('CP0001191/directors')
+    sinon.stub(axios, 'get').withArgs('CP0001191/directors?date=2019-04-01')
```

Comment:
If this file is being touched, I'd love to see the three lines of "data-app" code from, say, RegisteredOfficeAddress (lines 17-19) added to make the console output a little bit quieter.

Reply:
Yay!